portfolio/src/main/webapp/script.js

```diff
@@ -149,23 +147,71 @@ function deleteData(data) {
 // Creates a Map with Markers of Chick fil A locations in Tennessee
 function createMap() {
     const map = new google.maps.Map(
-    document.getElementById('map'),
-    {center: {lat: 35.832308, lng: -86.317786}, zoom: 8});
+        document.getElementById('map'), {
+            center: {
+                lat: 35.832308,
+                lng: -86.317786
+            },
+            zoom: 8
+        });
 
     var iconBase = 'https://maps.google.com/mapfiles/kml/shapes/';
-    
+
     const fiskUniversityMarker = new google.maps.Marker({
-    position: {lat: 36.1663542, lng:  -86.8031514},
-    map: map,
-    title: 'My College Campus',
-    icon: iconBase + 'parking_lot_maps.png'
+        position: {
+            lat: 36.1663542,
+            lng: -86.8031514
+        },
+        map: map,
+        title: 'My College Campus',
+        icon: iconBase + 'parking_lot_maps.png'
     });
 
     fetch('/chicFilA-data').then(response => response.json()).then((chicFilACoordinates) => {
-    chicFilACoordinates.forEach((chicFilACoordinate) => {
-      new google.maps.Marker(
-          {position: {lat: chicFilACoordinate.lat, lng: chicFilACoordinate.lng}, map: map});
+        chicFilACoordinates.forEach((chicFilACoordinate) => {
+            new google.maps.Marker({
+                position: {
+                    lat: chicFilACoordinate.lat,
+                    lng: chicFilACoordinate.lng
+                },
+                map: map
+            });
+        });
+    });
+}
+
+function fetchLogin() {
+    fetch('/login').then(response => response.json()).then((LoginData) => {
```

Comment:
why are some variables in this function starting with an uppercase letter? I think javascript conventions are that variables start with lowercase?

Reply:
you're right. Changed that